tests/async/test_browsertype_connect.py

```diff
@@ -12,11 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
+
 import pytest
 
 from playwright.async_api import BrowserType, Error
 from tests.server import Server
 
+pytestmark = pytest.mark.skipif(
```

Comment:
what kind of warning is that?

Reply:
https://github.com/aaugustin/websockets/issues/1029